portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,87 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', () => {
+  const navBar = document.getElementById('nav-bar');
+
+  Array.from(navBar.children).forEach(child => {
+    child.addEventListener('click', () => {
+      const targetId = child.getAttribute('data-target-id');
+      const targetElement = document.getElementById(targetId);
+      showContent(targetElement);
+    });
+  });
+});
+
+/* Displays the content title and text in the content container */
+function showContent(element) {
+  const contentContainer = document.getElementById('content-container');
+
+  Array.from(contentContainer.children).forEach(child => {
+    child.classList.toggle('hidden', true);
+  });
+
+  element.classList.toggle('hidden', false);
+}
+
+showContent(document.getElementById("about-me"));
```

Comment:
I might be missing something, but is this in the right place?

Reply:
I created this line so the page shows the "about-me" section on page load. The reason it's in this spot is because it uses showContent() so I put it right after that function.

Reply:
Instead of doing this here. Just remove the "hidden" class in the html for the "about-me" section.